plugins/email_service/app/services/service_layer/email_service_service.rb

```diff
@@ -1,14 +0,0 @@
-# frozen_string_literal: true
-
-module ServiceLayer
```

Comment:
Why did you remove the email service? You don't need Cronus anymore?

Reply:
restored.